src/form/CheckBox.tsx

```diff
@@ -22,7 +25,10 @@ export const CheckBox: React.FC<CheckBoxProps> = (props) => {
     form,
     onFocus,
     onBlur,
-    readOnly
+    readOnly,
+    validated,
+    errorMessage,
+    htmlTitle,
```

Comment:
I wonder why you validated the form checkbox in the other Checkbox component, not in this component? This component directly belongs to the form why the other checkbox can used in other places. If you want to reuse the checkbox validation in other places outside of the form as well then it makes sense though.

Reply:
To do it in the same compomnent as where the  <input/> element is defined is easiest. It prevents me from needing to pas refs acros components.
Also I am at the moment not sure if any user input that can be invalid should ever be requested outside a form element.
So this means I could use the checkbox outside a form, but only if a invalid state is not possible (for example like in the list, where any checkbox being toggled on/off is always valid)